src/languages/en.js

```diff
@@ -85,6 +85,9 @@ export default {
         noResultsFound: 'No results found',
         timePrefix: "It's",
         conjunctionFor: 'for',
+        today: 'Today',
+        tomorrow: 'Tomorrow',
+        yesterday: 'Yesterday',
```

Comment:
Also, we might want to make these lowercase. Looks odd here, whereas it looks good in slack

<img width="783" alt="Screenshot 2023-05-16 at 10 50 27 AM" src="https://github.com/Expensify/App/assets/32012005/a1b76b2e-f833-46d7-ac68-cc674084235a">

Reply:
Agreed, pushed a fix!